commerce-cloud-code/app_yourshophere/api/ProductSearchHit.js

```diff
@@ -19,40 +19,43 @@ exports.get = function get(apiHit, config) {
 
     Object.defineProperty(instance, 'tileUrl', {
         get: function getTileUrl() {
```

Comment:
factually this now returns a wainclude string, not sure if this is way to go but if yes we can find a good name